website/content/en/docs/Getting started/role-assignment.md

```diff
@@ -82,6 +84,23 @@ To enable fine-grained control on which user-assigned identity the cluster has a
 az role assignment create --role "Managed Identity Operator" --assignee <ID>  --scope /subscriptions/<SubscriptionID>/resourcegroups/<IdentityResourceGroup>/providers/Microsoft.ManagedIdentity/userAssignedIdentities/<IdentityName>
 ```
 
+## User-assigned managed identities for self-managed clusters
+
+If you deploy yourself the VMs and install kubernetes, you'll need to assign the user-assigned managed identity to the underlying VMs.

```

Comment:
```suggestion
If you deploy the VMs and install Kubernetes instead of using tools like [aks-engine](https://github.com/Azure/aks-engine) or [capz](https://github.com/kubernetes-sigs/cluster-api-provider-azure), you'll need to assign the user-assigned managed identity to the underlying VMs.

```